src/views/Vendor/Vendor.jsx

```
@@ -198,272 +243,312 @@ const Vendor = () => {
+            </p> {/* Added custom width and ensured the container is centered */}
+
+            <div className="flex gap-3 mt-3">
+              <button
+                style={{
+                  width: "90px",
+                  border: "1px solid black",
+                  borderRadius: "10px",
+                  padding: "8px",
+                }}
+                onClick={() => handleAddVendor("SUPPLIER")}
+              >
```

Biome

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)